client/webserver/api.go

```diff
@@ -263,13 +263,42 @@ func (s *WebServer) apiRegister(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, simpleAck(), s.indent)
 }
 
+// bondsFeeBuffer is a caching helper for the bonds fee buffer. Values for a
+// given asset are cached for 45 minutes. These values are meant to provide a
+// sensible but well-padded fee buffer for bond transactions now and well into
+// the future, so a long expiry is appropriate.
+func (s *WebServer) bondsFeeBuffer(assetID uint32) (feeBuffer uint64, err error) {
+	// (*Core).BondsFeeBuffer returns a fresh fee buffer based on a current (but
+	// padded) fee rate estimate. We assist the frontend by stabilizing this
+	// value for up to 45 minutes from the last request for a given asset. A web
+	// app could conceivably do the same, but we'll do this here between the
+	// backend (Core) and UI so that a webapp does not need to employ local
+	// storage/cookies and associated caching logic.
```

Comment:
My thinking was initially to use front end logic too, but the webserver cache is probably better. We are leaving rpcserver hanging for now.

Reply:
Correct re: the rpcserver.  However, changing responses from BondsFeeBuffer doesn't really matter for rpc since an rpc user would just record the value they obtained on their first call to BondsFeeBuffer and pass that on to PostBond, whereas on the frontend there's a the possibility to make the UI fetch a new value and you've lost the original.